pom.xml

```diff
@@ -136,6 +137,11 @@
                 <scope>test</scope>
             </dependency>
             <dependency>
+                <groupId>net.spals.midas</groupId>
```

Comment:
Looks like you can [use wildcards in an exclusion](http://stackoverflow.com/questions/547805/exclude-all-transitive-dependencies-of-a-single-dependency/7556707#7556707)